plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -306,7 +305,8 @@ def forward(self, x, scales=[]):
             If downsample is true, representation is an OrderedDict of the coefficients.
 
         """
-        self.pyr_coeffs = OrderedDict()
```

Comment:
So we're no longer storing pyr_coeffs in the class.  I'm basically on board, but can you walk me through the reasons why you decided that?

Reply:
it's primarily for when using the pyramid with large batches to limit the memory usage when we have for example the coefficients stored as a dictionary and then we return as a tensor in the non-downsampled version, the coeffs were stored in two places which doubles the memory. We thought the easiest thing would just be to let the module return the coeffs and the user can do whatever with it and delete it when necessary and that way the pyramid object is low-memory itself.